src/enclave-agent/Cargo.toml

```diff
@@ -13,7 +13,7 @@ clap = "2.33.3"
 # logger module
 env_logger = "0.10.0"
 
-image-rs = { git = "https://github.com/confidential-containers/guest-components.git", default-features = false, rev = "fe8fb1a" }
+image-rs = { git = "https://github.com/piotrpalcz/image-rs.git", default-features = false, branch = "random_key_2" }
```

Comment:
Just a note for later reference that we want to update this after https://github.com/confidential-containers/guest-components/pull/385 is merged

Reply:
to reflect that, I moved this PR to Draft.